chipy_org/apps/job_board/views.py

```diff
@@ -180,14 +180,12 @@ def get_queryset(self):
 
 
 class JobPostList(ListView):
-
     model = JobPost
     context_object_name = "job_posts"
     template_name = "job_board/job_post_list.html"
+    ordering = ("-is_sponsor", "-approval_date")
```

Comment:
Please remove line 186 here if you put the ordering for the queryset in the 'ApprovedAndActive' function in models.py